### PR TITLE
Documentation Improvements and Various Fixes

### PR DESCRIPTION
**Pull request by warmstarter**
base level documentation updates
included the html help
manpages and /help file are now fully spellchecked
manpages now build
TFVER added to configure as short form of TFVERSION
EINGPROGRESS crash bug patched
//command instead of ///command sends /command to MU
error_prefix is now customizable
OPTION102 no longer oddly capitalized in places

### CODE REVIEW

File: src/varlist.h [hunk 1]
```diff
@@ -35,11 +35,6 @@ varstrx(VAR_LC_CTYPE,	"LC_CTYPE",	NULL,		ch_locale)
 varstrx(VAR_LC_TIME,	"LC_TIME",	NULL,		ch_locale)
 #endif /* HAVE_SETLOCALE */
 varstr (VAR_MAIL,	"MAIL",		NULL,		ch_mailfile)
-#if ENABLE_OPTION102
```
Comment (kruton) [inline src/varlist.h:38]: Did you want to remove this?

File: src/socket.c [hunk 2]
```diff
@@ -1446,8 +1446,12 @@ static void setupnextconn(Sock *sock)
 {
     struct addrinfo *ai, *next = sock->addr;
 
-    if (sock->fd >= 0)
-	close(sock->fd);
+    if (sock->fd >= 0) {
+        FD_CLR(sock->fd, &readers);
+        FD_CLR(sock->fd, &writers);
+	      close(sock->fd);
```
Comment (kruton) [inline src/socket.c]: Whitespace issues?

File: src/output.c [hunk 3]
```diff
@@ -2984,7 +2984,7 @@ int clear_more(int new)
 	    nexttop(display_screen);
 	if (scrolled < new) {
 	    display_screen->paused = 0;
-	    display_screen->outcount = new - scrolled;
+            display_screen->outcount = new - scrolled;
```
Comment (kruton) [inline src/output.c]: Whitespace issues?

File: src/globals.h [hunk 4]
```diff
@@ -151,7 +151,6 @@ enum Vars {
 #define getstdvar(id)	((char*)         (strvar(id) ? strvar(id)->data : NULL))
 
 #define MAIL		getstdvar(VAR_MAIL)
-#define OPTION102	getintvar(VAR_OPTION102)
```
Comment (kruton) [inline src/globals.h:154]: Did you mean to remove this?

File: src/globals.h [hunk 5]
```diff
@@ -221,6 +221,7 @@ enum Vars {
 #define mprefix		getstrvar(VAR_mprefix)
 #define oldslash	getintvar(VAR_oldslash)
 #define optimize_user	getintvar(VAR_optimize)
+#define option102	getintvar(VAR_option102)
```
Comment (kruton) [inline src/globals.h:224]: Did you mean to remove this?

**Review by kruton — CHANGES_REQUESTED** [review]

File: src/varlist.h (hunk 1 above)
Comment (warmstarter) [reply src/varlist.h:38]: Yeah, I actually moved it, it's just showing up weird I guess. I confirmed and it's ok in the file.

There was no reason for it to be under VAR_OPTION102, so it's now under VAR_option102.

File: src/socket.c (hunk 2 above)
Comment (warmstarter) [reply src/socket.c]: Good catch, Github's web-merge is apparently something to be careful with in that regard. Just pushed corrections.

File: src/output.c (hunk 3 above)
Comment (warmstarter) [reply src/output.c]: Good catch, Github's web-merge is apparently something to be careful with in that regard. Just pushed corrections.

**Comment by warmstarter** [comment]
It's saying Changes requested, but I think that's the stuff I already did? I'm a bit new to github, so mostly following things, but just wanted to make sure that there's not something I'm missing.

Also, great work on figuring out that decode-attr so quick. I at least got to learn some gdb thanks to it.

**Comment by warmstarter** [comment]
Seems to be still a few warnings on the build test, but they were ones that apparently existed before. I solved the ones I knew off hand, but the others I would have to look into it at some point later.

Also, the build test is really cool, great job setting that up.

**Comment by warmstarter** [comment]
That buffer limit I just added is a lot higher, but It's been working for others and had been signed-off as part of a patch by some Linux Distribution people. I made it configurable though so we don't have a magic number there.

**Comment by kruton** [comment]
> It's saying Changes requested, but I think that's the stuff I already did? I'm a bit new to github, so mostly following things, but just wanted to make sure that there's not something I'm missing.
> 
> Also, great work on figuring out that decode-attr so quick. I at least got to learn some gdb thanks to it.

I think you'll have to split this pull request into multiple things. It's getting to be impossible to review. If you could split it into topic-focused pull requests, I'll be able to review it better than having a hundred commits on one branch that cover a variety of topics.

**Comment by warmstarter** [comment]
> > It's saying Changes requested, but I think that's the stuff I already did? I'm a bit new to github, so mostly following things, but just wanted to make sure that there's not something I'm missing.
> > Also, great work on figuring out that decode-attr so quick. I at least got to learn some gdb thanks to it.
> 
> I think you'll have to split this pull request into multiple things. It's getting to be impossible to review. If you could split it into topic-focused pull requests, I'll be able to review it better than having a hundred commits on one branch that cover a variety of topics.

How do I do that?

**Comment by kruton** [comment]
Make things into different git branches that cover only a specific topic and start an individual pull request for each of the topics.

**Comment by kruton** [comment]
Just wondering if you are still planning on splitting this?

**Comment by kruton** [comment]
Ah, it looks like you unforked this repository. I'll just go ahead and close this.

**Pull request closed without merging**